reframe/frontend/autodetect.py

```diff
@@ -177,6 +185,26 @@ def _emit_script_for_pip(job, env):
                 _log_contents(job.stdout)
                 _log_contents(job.stderr)
                 topo_info = json.loads(_contents('topo.json'))
+        else:
+            with _copy_reframe(prefix) as (dirname, use_pip):
```

Comment:
Do we need we a copy of reframe for pip? And does it make sense to make this available also for custom commands?